manifests/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -341,6 +341,11 @@ spec:
                 network support should be disabled. If unset, this property defaults
                 to 'false' and multiple network support is enabled.
               type: boolean
+            enableConnectivityChecks:
```

Comment:
(this will need to be updated in `openshift/api` as well)

Reply:
yep

Reply:
done